CMakeLists.txt

```diff
@@ -257,6 +257,7 @@ IF(NOT CYCLUS_DOC_ONLY)
 
     # include all the directories we just found
     INCLUDE_DIRECTORIES(${RECYCLE_INCLUDE_DIRS})
+    INCLUDE_DIRECTORIES(src src/toolkit src/toolkit/pyroprocessing)
```

Comment:
I don't think it is necessary to add `src/toolkit` and `src/toolkit/pyroprocessing`:
as you added `toolkit` in `src/CMakelists.txt`:
the main `CMakeLists.txt` will include `src` which will trigger the processing of `src/CMakeLists.txt` which includes `src/toolkit` so it will trigger the processing of `src/toolkit/CMakeLists.txt` and so on...

Reply:
That makes sense, I've just tested that and to get the includes to work properly I have to add the path: i.e. in diverter.h `#include "pyroprocessing/process.h"`. Is this better convention?

Reply:
Actually, I run into a build issue with pyre_tests when doing this. The test file cannot find the pyroprocessing directory

`In file included from /home/gtw2/recycle/build/recycle/pyre_tests.cc:1:0:
/home/gtw2/recycle/src/pyre.h:6:39: fatal error: pyroprocessing/pyre_volox.h: No such file or directory
 #include "pyroprocessing/pyre_volox.h"
`